kieker-common/src-gen/kieker/common/record/flow/AbstractEvent.java

```diff
@@ -24,7 +24,7 @@
 
 /**
  * @author Jan Waller
- * API compatibility: Kieker 1.15.0
```

Comment:
Is it intended to change this? If it is changed, it should be changed to 2.0.0 imho.

Reply:
Yes it should be upgrade to 2.0.0 I'll regenerate the classes.